### PR TITLE
Do not return error when getting cpu/memory capacity of node

### PR DESCRIPTION
**Pull request by losipiuk**

### CODE REVIEW

File: cluster-autoscaler/core/utils_test.go [hunk 1]
```diff
@@ -573,38 +573,51 @@ func TestConfigurePredicateCheckerForLoop(t *testing.T) {
...
 	assert.Equal(t, int64(2048), memory)
 
-	node.Status.Capacity = apiv1.ResourceList{}
+	// if we have no cpu/memory defined in capacity we expect getNodeCoresAndMemory to return 0s
+	nodeWithMissingCapacity := BuildTestNode("n1", 1000, 2*MB)
+	nodeWithMissingCapacity.Status.Capacity = apiv1.ResourceList{}
 
-	_, _, err = getNodeCoresAndMemory(node)
-	assert.Error(t, err)
+	cores, memory = getNodeCoresAndMemory(node)
```
Comment (aleksandra-malinowska) [inline cluster-autoscaler/core/utils_test.go]: s/node/nodeWithMissingCapacity

File: cluster-autoscaler/core/utils_test.go [hunk 2]
```diff
@@ -573,38 +573,51 @@ func TestConfigurePredicateCheckerForLoop(t *testing.T) {
...
 
-	memory, err := getNodeResource(node, apiv1.ResourceMemory)
-	assert.NoError(t, err)
+	memory := getNodeResource(node, apiv1.ResourceMemory)
 	assert.Equal(t, int64(2*MB), memory)
 
-	_, err = getNodeResource(node, "custom resource")
-	assert.Error(t, err)
+	unknownResourceValue := getNodeResource(node, "unknown resource")
+	assert.Equal(t, 0, unknownResourceValue)
```
Comment (aleksandra-malinowska) [inline cluster-autoscaler/core/utils_test.go]: int64(0)

File: cluster-autoscaler/core/utils_test.go (hunk 1 above)
Comment (losipiuk) [reply cluster-autoscaler/core/utils_test.go]: Should have run those tests .... sorry :/

**Pull request merged**